src/nvim/mbyte.c

```diff
@@ -704,12 +704,14 @@ bool utf_composinglike(const char_u *p1, const char_u *p2)
   return arabic_combine(utf_ptr2char(p1), c2);
 }
 
-/*
- * Convert a UTF-8 byte string to a wide character. Also get up to MAX_MCO
- * composing characters.
- *
- * @param [out] pcc: composing chars, last one is 0
- */
+/// Convert a UTF-8 string to a wide character
+///
+/// Also gets up to #MAX_MCO composing characters.
+///
+/// @param[out]  pcc  Location where to store composing characters. Must have
+///                   space at least for #MAX_MCO + 1 elements.
```

Comment:
@ZyX-I  , why `#MAX_MCO + 1` instead of `#MAX_MCO` elements. It looks like the terminating `NUL` is not written if at least `MAX_MCO` composing characters are in the string.

Reply:
I suspect I was going to refactor that into a saner version which has terminating zero always, but got distracted.